app/controller/command/commands/team.py

```diff
@@ -650,3 +669,30 @@ def refresh_all_team(self):
             self.facade.store(team_all)
         else:
             logging.error(f'Could not create {all_name}. Aborting.')
+
+    def refresh_drive_permissions(self):
+        """
+        Refresh Google Drive permissions based on user role. If no GCP client
+        is provided, this function is a no-op.
+        """
+
+        if self.gcp is None:
+            logging.debug("GCP not enabled, skipping drive permissions")
+            return
+
+        all_teams: List[Team] = self.facade.query(Team)
+        for t in all_teams:
+            if len(t.folder) == 0:
+                continue
+
+            emails: List[str] = []
+            for m in t.members:
+                if len(m.email) > 0:
+                    emails.append(m.email)
+
+            if len(emails) > 0:
+                logging.info("Synchronizing permissions for "
+                             + f"{t.github_team_name}'s folder ({t.folder}) "
+                             + f"to {emails}")
+                self.gcp.set_drive_permissions(
+                    t.folder, t.github_team_name, emails)
```

Comment:
You may want to run a more specialized version of this function, specific to the team that called with a `--folder`, to sync permissions for that team's team members only.

Something similar to what happens when we do `/rocket user edit --github` and we do an invite for that github user to join our organization.

Reply:
Do you think it's necessary at the moment? Happy to implement this in a follow-up instead :)